source/_lovelace/alarm-panel.markdown

```diff
@@ -23,22 +23,22 @@ entity: alarm_control_panel.alarm
 {% configuration %}
 type:
   required: true
-  description: alarm-panel
+  description: "`alarm-panel`"
   type: string
 entity:
   required: true
-  description: "Entity ID of `alarm_control_panel` domain"
+  description: "Entity ID from `alarm_control_panel` domain."
```

Comment:
Incorrect. `of` was the right naming here. It is a type of. From would have been correct if provided by another integration, but that is not the context in this case.